repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -418,17 +418,20 @@ def gather_target_repositories(context, indata):
     if missing_custom_repoids:
         raise StopActorExecutionError(
             message='Some required custom target repositories are not available.',
-            details={'hint': (
-                ' The most probably you are using custom or third party actor'
-                ' that produces CustomTargetRepository message or you did a typo'
-                ' in one of repoids specified on command line for the leapp --enablerepo'
-                ' option.'
-                ' Inside the upgrade container, we are not able to find such'
-                ' repository inside any repository file. Consider use of the'
-                ' custom repository file regarding the official upgrade'
-                ' documentation or check whether you did not do a typo in any'
-                ' repoids you specified for the --enablerepo option of leapp.'
-                )
+            details={
+                'hint': (
```

Comment:
Hmm, while this is a step in the right direction quite frankly even though I know all the details how this is supposed to work the current message isn't really helpful. Besides that, this is going to look like a soup of text both in the text report as well as in Cockpit/Satellite.

This should be refactored to use `create_report` with an `inhibitor` and properly linked `RelatedResources`.
Also, in practice, it's much more likely that I'm seeing this message as a result of `--enablerepo` than custom actors.

Lastly, if this is really caused by a custom actor we should be able to print the name/path of the actor.

Let's use this PR as a draft, I've created OAMG-5519 to flesh out the desired state.

Reply:
I agree with Pavel. However, I think we can merge this as it will be easy to backport it. For master, we can continue to improve to use ```inhibitor``` and proper linked ```RelatedResources```

Reply:
In this case, we cannot apply now inhibitor as this is breaking situation. I mean - if we want to handle this reaaly properly without an errors, we have to do much more changes. This was just a fast offer to resolve this now.

@shaded-enmity  this is not cause by any custom actor. It's caused e.g. by typos in CLI or improper Satellite setup.